filebeat/input/tcp/config.go

```diff
@@ -40,6 +40,7 @@ var defaultConfig = config{
 	Config: tcp.Config{
 		Timeout:        time.Minute * 5,
 		MaxMessageSize: 20 * humanize.MiByte,
+		MaxConnections: 10,
```

Comment:
What are the effects of this default value on a typical node? Are we confident that this is high enough not to artificially bottleneck a node that was previously handling a higher load? I'd be happier if this value was higher, or if the default was "no maximum"

Reply:
Good point! Updated the configuration to be unlimited by default.